test/e2e/snaps/enums.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   TEST_SNAPS_WEBSITE_URL:
-    'https://metamask.github.io/snaps/test-snaps/0.38.0-flask.1/',
+    'https://metamask.github.io/snaps/test-snaps/0.39.1-flask.1/',
```

Comment:
```suggestion
    'https://metamask.github.io/snaps/test-snaps/1.0.0/',
```

Reply:
Oh I must have forgotten to change this in my PR 😅